src/Sylius/Bundle/AdminBundle/Tests/Security/AdminFirewallTest.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\AdminBundle\Tests\Security;
+
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+
+class AdminFirewallTest extends WebTestCase
```

Comment:
AdminFirewallTest should be final because Tests should not be extendable

```suggestion
final class AdminFirewallTest extends WebTestCase
```